3 - PB/Documentazione esterna/Verbali/24-02-22/24-02-22.typ

```diff
@@ -0,0 +1,83 @@
+#import "/template.typ": *
+
+#show: project.with(
+  date: "22/02/24",
+  subTitle: "Aggiornamento sullo stato dei lavori",
+  docType: "verbale",
+  authors: (
+    "Todesco Mattia",
+  ),
+  externalParticipants : (
+    (name: "Bassani Matteo", role: "Referente aziendale"),
+  ),
+  timeStart: "15:20",
+  timeEnd: "16:10",
+  location: "Zoom",
+);
+
+= Ordine del giorno
+
+- Aggiornamento sullo stato dei lavori;
+- Feedback su modalità e strumenti per il testing;
+- Feedback sul deploy dell'applicazione;
+- Presentazione bozza diagramma ER;
+- Meeting futuri.
+
+== Aggiornamento sullo stato dei lavori
+
+Il Proponente è stato aggiornato in maniera esaustiva sull'esito del duplice colloquio di RTB sostenuto dal gruppo.
```

Comment:
~in maniera esaustiva~. Non possiamo dirlo noi se è stata esaustiva. Limitiamoci a dire che lo abbiamo informato.